datetime_tz/win32tz_map.py

```diff
@@ -0,0 +1,110 @@
+'''Map between Windows and Olson timezones taken from http://www.unicode.org/repos/cldr/trunk/common/supplemental/windowsZones.xml
```

Comment:
If I remove it, the sdist build fails as it tries to import it before it's created it.  Would you prefer solving that with a conditional import, or import just before using it?